### ISSUE TITLE
'Manager' object has no attribute 'restrict' after adding script count widget in dashboard

### ISSUE THREAD

**Issue by markkuleinio** [netbox-community/netbox#14933]
### Deployment Type

Self-hosted

### NetBox Version

v3.7.1

### Python Version

3.11

### Steps to Reproduce

In the dashboard:

1. Unlock Dashboard
2. Add Widget
3. Widget type = Object Counts
4. Models = Extras > Scripts
5. Save

### Expected Behavior

There is a widget in the dashboard that shows the number of scripts

### Observed Behavior

![image](https://github.com/netbox-community/netbox/assets/25136274/f0998512-acbe-46a5-bcd8-b100a846beb1)

**<class 'AttributeError'>
'Manager' object has no attribute 'restrict'**

Dashboard reset (/extras/dashboard/reset/) is required to be able to use the home page again.

**Comment by llamafilm** [comment]
I just ran into the same problem.  I was able to fix it in nbshell like this, after inspecting the `User` to find the right ID.
```
>>> dash = Dashboard.objects.get(id=16)
>>> dash.delete_widget(id='a04674a6-6344-46f8-b677-6a9e6443277a')
>>> dash.full_clean()
>>> dash.save()
```

**Comment by arthanson** [comment]
blocked by #9856

**Comment by arthanson** [comment]
This will be fixed in NB 4.0 via #14438
